scripts/list-routes.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+
+require 'json'
+require 'set'
+
+routes_by_domain_url = {}
+
+next_url = '/v2/routes?results-per-page=100'
+while next_url
+  page = JSON.parse `cf curl '#{next_url}'`
+  page['resources'].map do |resource|
+    host = resource.dig('entity', 'host')
+    unless host.empty?
```

Comment:
In London this excludes 51 routes which do have an empty `host` field. Most seem to be using path-based routing. 😞

Reply:
```suggestion
```